db/migrate/20220603145240_change_user_gender_enum.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeUserGenderEnum < ActiveRecord::Migration[7.0]
+  def up
+    execute "ALTER TABLE users ALTER gender DROP DEFAULT;"
+    change_column :users, :gender, :integer, using: 'gender::integer'
```

Comment:
What is this `using` for?

Reply:
because without using when I try to run the migration, it gave an error

<img width="742" alt="Screen Shot 2022-06-08 at 12 46 57" src="https://user-images.githubusercontent.com/87084375/172660644-a32c1c76-e292-4767-9af8-e2a59fd45986.png">